### ISSUE TITLE
Slide templates for talks on Dask

### ISSUE THREAD

**Issue by GenevieveBuckley** [dask/marketing#10]
Perhaps it would be useful to have slide templates available for talks, etc. The style guide page is nice, but it's kind of a hassle picking out the right hex colours, fonts, etc. from scratch and I'd prefer to have a template available.

I put these google slide themes together for myself, and could share them or something similar here. We could put something together for powerpoint as well.

* Dark theme: https://docs.google.com/presentation/d/1jkpmB83ijzaOmeaKhJtmcJpnLDCyD1bWXxLJx-ApMdM/template/preview
* Light theme: https://docs.google.com/presentation/d/1i-lHFwfQnFyKd-1XsthELmUZW11a7jXZrBUkLgBr15k/template/preview

**Comment by mrocklin** [comment]
I typically take one of my existing decks, like https://dask.org/slides , make a copy, and go from there.  I admit though that this probably isn't fully accessible to people who haven't at some point asked me for a copy.  

Turning this (or better yet learning from and improving upon by cleaning up) into a template sounds great to me.  We can probably also ask some of the designers that Coiled works with to help out here a bit.  I'll ping folks there.

**Comment by GenevieveBuckley** [comment]
Yeah, that would be ideal. There's no way for me to make a copy of that slide deck from that link though - at least, I tried that first but couldn't do it.
